public/pages/createDetector/containers/utils/detectorToFormik.ts

```diff
@@ -41,6 +41,6 @@ export function detectorToFormik(ad: Detector): ADFormikValues {
     timeField: ad.timeField,
     detectionInterval: get(ad, 'detectionInterval.period.interval', 10),
     windowDelay: get(ad, 'windowDelay.period.interval', 0),
-    shingleSize: get(ad, 'shingleSize', SHINGLE_SIZE),
+    shingleSize: get(ad, 'shingleSize', SINGLE_ENTITY_SHINGLE_SIZE),
```

Comment:
Can we use the `getShingleSizeFromObject` helper fn you made, in the case that a HC detector is passed here? Could consider if there is a `categoryField` then `isHCDetector` would be `true`.